biojava-core/src/test/resources/org/biojava/nbio/core/sequence/GenbankProxySequenceReader/152970917.gb

```diff
@@ -0,0 +1,70 @@
+LOCUS       YP_001336026             324 aa            linear   CON 16-DEC-2014
```

Comment:
See my comment above about where to store these resource files. The convention should be the same package name as where the test class is (I know we did not always follow that in the past, but we should be doing it at least for new tests)